TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -415,14 +415,18 @@ private void WritePSMResults()
                 WritePsmsToTsv(psmsForThisFile, writtenFile, Parameters.SearchParameters.ModsToWriteSelection);
                 SucessfullyFinishedWritingFile(writtenFile, new List<string> { Parameters.SearchTaskId, "Individual Spectra Files", group.First().FullFilePath });
                 Parameters.SearchTaskResults.AddNiceText("Target PSMs within 1% FDR in " + strippedFileName + ": " + psmsForThisFile.Count(a => a.FdrInfo.QValue < .01 && a.IsDecoy == false));
-                Parameters.SearchTaskResults.AddNiceText("MS2 spectra in " + strippedFileName + ": " + Parameters.NumMs2SpectraPerFile[strippedFileName][0]);
-                Parameters.SearchTaskResults.AddNiceText("Number of precursor species fragmented in " + strippedFileName + ": " + Parameters.NumMs2SpectraPerFile[strippedFileName][1]);
 
                 var writtenFileForPercolator = Path.Combine(Parameters.OutputFolder, strippedFileName + "_forPercolator.tsv");
                 WritePsmsForPercolator(psmsForThisFile, writtenFileForPercolator);
                 SucessfullyFinishedWritingFile(writtenFileForPercolator, new List<string> { Parameters.SearchTaskId, "Individual Spectra Files", group.First().FullFilePath });
             }
             foreach (var group in PsmsGroupedByFile)
+            {
```

Comment:
I like this idea because it's nicer looking than the current version but I think it's better to have one line per piece of information here. It's a bit weird to have # MS2 spectra and # fragmented precursors on the same line, those seem unrelated to me. I think it would also look better if it came either before or after the results instead of in the middle. 
maybe like this: 

MS2 spectra in 01-23-17_YL_120mins_top10: 37687
MS2 spectra in 01-23-17_YL_150mins_top15: 50044
Precursors fragmented in 01-23-17_YL_120mins_top10: 81922
Precursors fragmented in 01-23-17_YL_150mins_top15: 106039
Target PSMs within 1% FDR in 01-23-17_YL_120mins_top10: 26808
Target PSMs within 1% FDR in 01-23-17_YL_150mins_top15: 34343
Target peptides within 1% FDR in 01-23-17_YL_120mins_top10: 16992
Target peptides within 1% FDR in 01-23-17_YL_150mins_top15: 19562
Target protein groups within 1 % FDR in 01-23-17_YL_120mins_top10: 2475
Target protein groups within 1 % FDR in 01-23-17_YL_150mins_top15: 2743

Reply:
done